EnglishFilter/sections/specific.txt

```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+desishoot.fun##.islemag-content-right > .widget_custom_html
```

Comment:
Missed tag after `.islemag-content-right > `

Reply:
Okay, will add.

Reply:
`.islemag-content-right` doesn't need tag?

Reply:
That's not necessary for first element, we add tags after `>`.
Also we skip it, if element has different tags.
So, the rule becomes more accurate and convenient for our team to review.

Reply:
I got it, and the same goes for descendant combinator (space)?

Reply:
Yes, the same)